translations/de/module_de_lang.php

```diff
@@ -122,6 +122,7 @@
     'wd_last-name' => 'Nachname',
     'wd_maid' => 'MAID',
     'wd_manipulated' => 'manipuliert',
+    'wd_matched' => 'zugeordnet',
```

Comment:
Do we still need translation for "matched"?

Reply:
No, it should not be needed anymore.

Reply:
👍